RUFAS/routines/animal/ration/ration_driver.py

```diff
@@ -865,10 +878,15 @@ def get_NE_growth(
         float
             Net energy of feed i, Mcal/kg.
 
+        Notes
+        -----
+        Division of ME_item by kg_fed is to get the ME for each item on a per kg basis,
```

Comment:
```suggestion
        Division of ME_item by kg_fed is to get the metabolizable energy for each item on a per kg basis,
```
I'm not sure if this is correct or not, but expanding abbreviations like this would really help non-SMEs understand what is going on.